ipynb/pick_otus.ipynb

```diff
@@ -0,0 +1,228 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pick OTUs (for full and trimmed data) at approximately genus level resolution (97% similarity) using SortMeRNA, closed reference against Greengenes 13_8."
```

Comment:
Just to sanity check.

Can we assume that people looking at these notebooks have prior knowledge about OTU picking?

If not, it may be desirable to at least include a link.